### PR TITLE
Paddledet support

### PR DESCRIPTION
**Pull request by hardikdava**
# Description

This PR is in support for #187 

## Type of change

-   [x] New feature (non-breaking change which adds functionality)
-   [x] This change requires a documentation update

## How has this change been tested, please provide a testcase or example of how you tested the change?

Find colab [here](https://drive.google.com/file/d/1R8mLUFsoflwTK4PSfnOXmKtw7Keu2zlX/edit?pli=1)

## Docs

-   [ ] Docs updated? What were the changes:

### CODE REVIEW

**Comment by SkalskiP** [comment]
@hardikdava, do you want to ship it in this release?

**Comment by hardikdava** [comment]
@SkalskiP yeah, we can include it.

**Comment by SkalskiP** [comment]
@hardikdava, do you have some Colab I could use to test it?

File: supervision/detection/core.py [hunk 1]
```diff
@@ -354,6 +354,14 @@ def from_sam(cls, sam_result: List[dict]) -> Detections:
 
         return Detections(xyxy=xywh_to_xyxy(boxes_xywh=xywh), mask=mask)
 
+    @classmethod
+    def from_paddledet(cls, paddledet_result):
```
Comment (SkalskiP) [inline supervision/detection/core.py:358]: We will also need a docstring for this connector.

**Review by SkalskiP — CHANGES_REQUESTED** [review]

**Comment by hardikdava** [comment]
> @hardikdava, do you have some Colab I could use to test it?

[Here](https://colab.research.google.com/drive/1R8mLUFsoflwTK4PSfnOXmKtw7Keu2zlX#scrollTo=5ESh2QrCbbGn) you go.

**Comment by SkalskiP** [comment]
> Here

I don't have access :/

**Comment by hardikdava** [comment]
> > Here
> 
> I don't have access :/

Check now please.

**Comment by SkalskiP** [comment]
@hardikdava I'm unsure if I'm doing something wrong, but the Colab fails. I get an exception.

Screenshot 2023-07-21 at 12 17 11

**Comment by hardikdava** [comment]
Yeah, I am also getting same error. But it was working fine :laughing:

**Comment by SkalskiP** [comment]
@hardikdava would you have a bit of time to take a look at it?

**Comment by hardikdava** [comment]
@SkalskiP  I tried various installation but I could not do it. Let's keep this on next release.

**Comment by SkalskiP** [comment]
@hardikdava, no worries! 🙏🏻 I plan to close `0.12.0` release on Monday. I'm happy to move this one to `0.13.0` and have our full focus on things we can still close.

**Comment by hardikdava** [comment]
hello @SkalskiP , I fixed the colab test. Please test it with update colab: https://colab.research.google.com/drive/1R8mLUFsoflwTK4PSfnOXmKtw7Keu2zlX?usp=sharing

**Comment by SkalskiP** [comment]
@hardikdava I'm still trying to figure out what's going on, but I still doesn't work for me. 

Screenshot 2023-07-22 at 17 30 34

**Comment by onuralpszr** [comment]
> @hardikdava I'm still trying to figure out what's going on, but I still doesn't work for me.
> 
> Screenshot 2023-07-22 at 17 30 34

Let me take tackle this a little bit I think I have an idea......

**Comment by hardikdava** [comment]
@onuralpszr thanks. I like how we are helping each other with bugs and errors.

**Comment by onuralpszr** [comment]
@hardikdava  @SkalskiP  hurray :)) 

image

**Comment by onuralpszr** [comment]
Do this commands then re-import again

```bash
!wget http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb
!sudo dpkg -i libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb
```

**Comment by SkalskiP** [comment]
@hardikdava am I correct to assume we are ready to merge here?

**Comment by onuralpszr** [comment]
> @hardikdava am I correct to assume we are ready to merge here? 

Also, Did you able to run with my fix ?

**Comment by hardikdava** [comment]
> > @hardikdava am I correct to assume we are ready to merge here?
> 
> Also, Did you able to run with my fix ?

Thanks @onuralpszr. yeah, I was able to run it.

**Review by SkalskiP — APPROVED** [review]

**Comment by SkalskiP** [comment]
Hi @hardikdava! 👋🏻 Just tested your Colab. Looks good. Merging! 🔥

**Pull request merged**